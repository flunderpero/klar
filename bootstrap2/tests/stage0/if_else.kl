fn main():
    let a = 1
    let b = 2
    if a == 1:
        print("a is 1")
    end
    if a == b:
        print("a is b")
    else:
        print("a is not b")
    end
    if a == 1 and b == 2:
        print("a is 1 and b is 2")
    end
    if a == 2 or b == 2:
        print("a is 2 or b is 2")
    end
    -- if / else if / else
    if a == b => print("a is b")
    else => if a == 1 => print("a is 1")
    else => print("a is not 1")
end
