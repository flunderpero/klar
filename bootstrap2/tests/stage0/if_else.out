a is 1
a is not b
a is 1 and b is 2
a is 2 or b is 2
a is 1
